projects/commudle-admin/src/app/components/sidebar-menu/sidebar-menu.component.html

```diff
@@ -96,6 +96,9 @@
         <nb-accordion-item-body *ngIf="isBadgesAdmin">
           <a [routerLink]="['/sys-admin', 'badges']">Badges</a>
         </nb-accordion-item-body>
+        <nb-accordion-item-body>
+          <a [routerLink]="['/newsletter-list']">Newsletters</a>
```

Comment:
Let's move this to the default menu on top

Reply:
moved